<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="north_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="east_pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="west_pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teleport_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Teleport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="south_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>231</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="info_pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="mapDisplay">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="map_pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>313</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
